src/PFDV2/pfd/shared/PFDSimvarPublisher.tsx

```diff
@@ -0,0 +1,325 @@
+    potentiometer_captain: number;
+    potentiometer_fo: number;
+    pitch: number;
+    roll: number;
+    heading: number;
+    altitude: number;
+    speed: number;
+    alpha_prot: number;
+    onGround: number;
+    activeLateralMode: number;
+    activeVerticalMode: number;
+    fma_mode_reversion: number;
+    fma_speed_protection: number;
+    AThrMode: number;
+    ap_vs_selected: number;
+    approachCapability: number;
+    ap1Active: boolean;
+    ap2Active: boolean;
+    fmaVerticalArmed: number;
+    fmaLateralArmed: number;
+    fd1Active: boolean;
+    fd2Active: boolean;
+    athrStatus: number;
+    athrModeMessage: number;
+    machPreselVal: number;
+    speedPreselVal: number;
+    mda: number;
+    dh: number;
+    attHdgKnob: number;
+    airKnob: number;
+    vs_baro: number;
+    vs_inert: number;
+    sideStickX: number,
+    sideStickY: number,
+    fdYawCommand: number,
+    fdBank: number,
+    fdPitch: number,
+    v1: number,
+    vr:number,
+    fwcFlightPhase: number,
+    fmgcFlightPhase: number,
+    hasLoc: boolean;
+    hasDme: boolean;
+    navIdent: string;
+    navFreq: number;
+    dme: number;
+    navRadialError: number;
+    hasGlideslope: boolean;
+    glideSlopeError: number;
+    markerBeacon: number;
+    isAltManaged: boolean;
+    vMax: number;
+    targetSpeedManaged: number;
+    mach: number;
+    flapHandleIndex: number;
+    greenDotSpeed: number;
+    slatSpeed: number;
+    fSpeed: number;
+    transAlt: number;
+    transAltAppr: number;
+    groundTrack: number;
+    showSelectedHeading: number;
+    altConstraint: number;
+    trkFpaActive: boolean;
+    aoa: number;
+    groundHeadingTrue: number;
+    groundTrackTrue: number;
+    selectedFpa: number;
+    vfeNext: number;
+    ilsCourse: number;
+    metricAltToggle: boolean;
+    tla1: number;
+    tla2: number;
+    landingElevation: number;
+    tcasState: number;
+    tcasCorrective: boolean;
+    tcasRedZoneL: number;
+    tcasRedZoneH: number;
+    tcasGreenZoneL: number;
+    tcasGreenZoneH: number;
+    tcasFail: boolean;
+    engOneRunning: boolean;
+    engTwoRunning: boolean;
+    expediteMode: boolean;
+    setHoldSpeed: boolean;
+    vls: number;
+    alphaLim: number;
+    trkFpaDeselectedTCAS: boolean,
+    tcasRaInhibited: boolean,
+    groundSpeed: number,
+    radioAltitude1: number,
+    radioAltitude2: number,
+  }
+
+export enum PFDVars {
+    ColdDark = 'L:A32NX_COLD_AND_DARK_SPAWN',
+    elec = 'L:A32NX_ELEC_AC_ESS_BUS_IS_POWERED',
+    elecFo = 'L:A32NX_ELEC_AC_2_BUS_IS_POWERED',
+    potentiometer_captain= 'LIGHT POTENTIOMETER:88',
+    potentiometer_fo= 'LIGHT POTENTIOMETER:90',
+
+    pitch = 'L:A32NX_ADIRS_IR_1_PITCH',
+    roll = 'L:A32NX_ADIRS_IR_1_ROLL',
+    heading = 'L:A32NX_ADIRS_IR_1_HEADING',
+    altitude = 'L:A32NX_ADIRS_ADR_1_ALTITUDE',
+    speed = 'L:A32NX_ADIRS_ADR_1_COMPUTED_AIRSPEED',
+    alpha_prot = 'L:A32NX_SPEEDS_ALPHA_PROTECTION',
+    onGround = 'SIM ON GROUND',
```

Comment:
We have LGCIU discrete vars for this now... https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md#landing-gear-ata-32